scss/_variables.scss

```diff
@@ -237,6 +237,10 @@ $border-radius:               .25rem !default;
 $border-radius-lg:            .3rem !default;
 $border-radius-sm:            .2rem !default;
 
+$box-shadow-sm:               0 .25rem .5rem rgba(#1a1a1a, .2) !default;
+$box-shadow:                  0 .75rem 2rem rgba(#1a1a1a, .24) !default;
+$box-shadow-lg:               0 1.5rem 2.5rem rgba(#1a1a1a, .16) !default;
```

Comment:
Need to determine if `#1a1a1a` needs to be memorialized as a named color, or if this can be changed to an existing color already specified in our base design system.

Reply:
I'd generally recommend keeping what the component specs say for colors rather than diverging to what exists currently.   That will help us down the road when QA-ing visually against figma files.

Whether this becomes a brand-"named" color like malachite or something we decide on, no strong opinion other than I've always appreciated variables in past scss files.
 
For my own curiosity, what's the rationale behind using hex here over rgb?  
<img width="205" alt="Screen Shot 2020-07-08 at 3 57 38 PM" src="https://user-images.githubusercontent.com/1186377/86964231-c1f87580-c133-11ea-8973-00fb17201c0c.png">

Reply:
I can agree with that rationale.

Reason for the hex code is that's been our standard format for named colors; wanted to put this unnamed color in that format for future extraction to a named variable.

Reply:
Cool I'm down with consistency.  I like that we can mix hex and alpha so it gets us everything we need

Reply:
TIL [it's a Sass thing only](https://sass-lang.com/documentation/modules#rgba) - can't do this in regular CSS.